src/operation.ts

```diff
@@ -116,7 +116,7 @@ export default class Operation {
 
     // Favor JSON if it exists
     types.forEach(t => {
-      if (t.match(/json/)) {
+      if (matchesMimeType.json(t)) {
```

Comment:
`matchesMimeType.json()` does the same as this, this change is just to be more consistent in how we handle content type matching.